variables.tf

```diff
@@ -237,7 +244,7 @@ variable "parameter_group_tags" {
 
 variable "create_replication_group" {
   type        = bool
-  default     = true
+  default     = false
```

Comment:
* You forgot to set `create_replication_group = true` **[here](https://github.com/itsyndicate/terraform-aws-elasticache/blob/main/examples/complete/terragrunt.hcl).**